### ISSUE TITLE
error for images: 'MediaManager' object has no attribute 'syncDelete'.

### ISSUE THREAD

**Issue by red5247** [c-okelly/anki-remote-decks#52]
Hi,

Just started using this and love it! Everything works great except the images in my google doc don't come through. I get this error: 'MediaManager' object has no attribute 'syncDelete'. I imagine I'm missing something obvious, but I can't figure out what it is. Thanks 
![image](https://user-images.githubusercontent.com/30841949/78568211-642c5880-77e7-11ea-9487-8370c6038802.png)

**Comment by c-okelly** [comment]
That is a strange error. Could you share the Google Doc with me so I can have a look and try an replicate?

I do don't want to post it publicly you can email me okellyconor@gmail.com

**Comment by c-okelly** [comment]
ok I'm able to replicate the error. It appears my add on no longer works on the latest version of Anki which is annoying. 

I will investigate and roll our a fix

**Comment by c-okelly** [comment]
One of the hooks I was using to add media to Anki has changed so will required updates.

The number of changes in the last few versions and how few hooks there are is worrying. I will try and have this fix done tomorrow

**Comment by c-okelly** [comment]
Following code does seem to work. I'm still testing backward compatibility though

```python
    def some_function(filename):
        try:
            self.media().syncDelete(filename)
        except AttributeError:
            self.media().trash_files([filename])
```

**Comment by c-okelly** [comment]
Issue should be resolved in the latest version v1.5.2. Just releated
